openapi-cli/src/test/resources/generators/schema/ballerina/default_non_required_field_schema.bal

```diff
@@ -0,0 +1,7 @@
+public type Pet record {
+    1|2|3 id = 2;
+    "red"|"green"|"blue" color = "green";
+    string name = "Name";
+    int age = 1;
+    string 'type?;
```

Comment:
Shall we add some boolean samples as well?

Reply:
added